contracts/MessengerRegistry.sol

```diff
@@ -88,6 +88,11 @@ contract MessengerRegistry is IMessengerRegistry {
         uint256 id = _messengers.length;
         _ownerMessengers[messengerOwner].push(id);
 
+        require(
+            precision % 100 == 0 && precision != 0,
```

Comment:
Must use SafeMath operations to  avoid overflow + extra gas consumption

See https://docs.openzeppelin.com/contracts/2.x/api/math#SafeMath-mod-uint256-uint256-string-